modules/gdscript/gdscript_cache.cpp

```diff
@@ -30,6 +30,8 @@
 
 #include "gdscript_cache.h"
 
+#include "core/error/error_list.h"
+#include "core/error/error_macros.h"
```

Comment:
These includes are not needed. They're technically correct, but until we decide on a policy to always include every single header directly used in a class (instead of relying on nested includes in other existing dependencies), this mostly adds noise to PRs and inconsistency in the codebase.
I guess you have an IDE setting responsible for this which would be good to disable for Godot work.